controller.go

```diff
@@ -272,6 +274,13 @@ func (c Controller) jsonHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	enc := json.NewEncoder(w)
+
+	clusterAckMsg := c.registry.clusterAck()
+	if clusterAckMsg != "" {
+		healthResults.Description = fmt.Sprintf("%s Cluster is acknowledged: %s", healthResults.Description, clusterAckMsg)
```

Comment:
The healthResults.Name is rather the healthcheck name, i.e. "COCO pre-prod cluster's health"